cookies/views.py

```diff
@@ -1179,3 +1180,41 @@ def resource_content(request, resource_id):
                 target += '?accessToken=' + auth_token
         return HttpResponseRedirect(target)
     return HttpResponse('Nope')
+
+def export_coauthor_data(request):
+    context = RequestContext(request, {})
+
+    graph = nx.Graph()
+
+    parent_id = request.GET.get('parent_collection', None)
+
+
+    if parent_id:
+        parent_collection = _get_collection_by_id(request, int(parent_id))
+        check_auth = authorization.check_authorization('view_collection', request.user, parent_collection)
+        if not check_auth:
```

Comment:
Wherever possible, avoid setting variables that are only used once. It simplifies debugging, and makes your code easier to read. In this case, you can do:

```python
if not authorization.check_authorization('view_collection', request.user, parent_collection):
    return HttpResponse('You do not have permission to view this collection', status=401)
```